### PR TITLE
Androidx update

### PR DESCRIPTION
**Pull request by parshav**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -15,6 +15,7 @@ android {
         versionCode 2
         versionName "1.0.1"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        multiDexEnabled true
```
Comment (benwicks) [inline app/build.gradle]: Why is this needed?

Comment (parshav) [reply app/build.gradle]: True, I added it in to try and remove an error I was having earlier, slipped to removed.
